lib/solidus_importer/processors/product_images.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  module Processors
+    class ProductImages < Base
+      def call(context)
+        @data = context[:data]
+        @entity = context[:entity]
+        context.merge!(check_data || save_images)
+      end
+
+      private
+
+      def check_data
+        if @data.blank?
+          { success: false, messages: 'Missing input data' }
+        elsif @data['Image Src'].blank?
+          {}
+        elsif !@entity || !@entity.is_a?(Spree::Product) || !@entity.valid?
+          { success: false, messages: 'Target entity must be a valid product' }
```

Comment:
As above, checking for the product's validity here seems out of scope.

Reply:
@blocknotes remember to fix this one 🎗️

Reply:
I fixed it here:
https://github.com/nebulab/solidus_importer/pull/8/commits/8173598e686c19a2f1b4b59e0e201a48cc7e9595

Reply:
Sure 😅 done now. Here:
https://github.com/nebulab/solidus_importer/pull/8/commits/85b45961869aa7f1b65acf1dfc2d64eba11958b3